{"abstract":[{"type":"text","text":"The certifcate could not be parsed."}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"invalidCertificate","kind":"identifier"}],"modules":[{"name":"FIDO2"}],"title":"PublicKeyCredentialError.invalidCertificate","symbolKind":"case","externalID":"s:5FIDO224PublicKeyCredentialErrorO18invalidCertificateyA2CmF","role":"symbol","roleHeading":"Case"},"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError"]]},"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/invalidCertificate"},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"invalidCertificate","kind":"identifier"}]}],"kind":"declarations"}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/fido2\/publickeycredentialerror\/invalidcertificate"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://com.ibm.verify.ios.sdk/documentation/FIDO2":{"type":"topic","role":"collection","kind":"symbol","title":"FIDO2","url":"\/documentation\/fido2","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","abstract":[]},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError":{"navigatorTitle":[{"text":"PublicKeyCredentialError","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError","title":"PublicKeyCredentialError","abstract":[{"type":"text","text":"A type that indicates when Fido2ApiClient encounters an error."}],"url":"\/documentation\/fido2\/publickeycredentialerror","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"PublicKeyCredentialError"}],"kind":"symbol"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError/invalidCertificate":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"invalidCertificate","kind":"identifier"}],"title":"PublicKeyCredentialError.invalidCertificate","type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/invalidCertificate","url":"\/documentation\/fido2\/publickeycredentialerror\/invalidcertificate","abstract":[{"text":"The certifcate could not be parsed.","type":"text"}],"role":"symbol","kind":"symbol"}}}